### DISCUSSION TITLE
NET::ERR_CERT_AUTHORITY_INVALID

### DISCUSSION THREAD

**Discussion by robertodc** [globaleaks/globaleaks-whistleblowing-software#3968] [Q&A]
Hi!
After installing the platform on Aruba cloud, when I access the platform I get a page which says: NET::ERR_CERT_AUTHORITY_INVALID.
Any help. please?

**Comment by evilaliv3** [comment]
Thank you for your question.

Actually when you setup the software globaleaks creates a self signed
certificate that you could use during first setup or when updating the
certificate.

To proceed with the setup you should access to the web interface, configure
a domain name and enable a certificate specific for your domain name.

If you do that the certificate error will disappear.

**Reply by robertodc** [reply]
Grazie.
Potresti dirmelo in italiano, per favore?
Il mio inglese è proprio basso basso......
Grazie.

**Reply by evilaliv3** [reply]
Grazie per la tua domanda.

Infatti, quando si configura il software, globalaks crea un certificato self-signed destinato ad essere usato durante la prima configurazione o durante l'aggiornamento del proprio certificato.

Per risolvere il problema che riscontri (l'errore sul certificato self signed) è necessario che tu acceda all'interfaccia web, configureun nome di dominio e attivano un certificato specifico per il tuo nome di dominio.

**Comment by robertodc** [comment]
Buonasera,
non ho ancora risolto il problema. Avevo girato la sua risposta al ns. tecnico, il quale aveva attivato la piattaforma su Aruba Cloud, e mi risponde in questo modo per eliminare il problema: "Sicuramente è una cosa a pagamento. Costerà un po' meno di cento euro all'anno credo. Il server che vi ho attivato non sembra averla questa funzionalità. Ho chiesto al gestore istruzioni su come fare. Sollecito la richiesta e le faccio sapere qualcosa."
Possibile dover pagare anche per togliere l'errore del certificato?
Forse è possibile agire in altro modo?
